specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink",
+          "itemName": "value"
+        }
+      }
+    },
+    "/chat/threads/{chatThreadId}": {
+      "patch": {
+        "tags": [
+          "Threads"
+        ],
+        "summary": "Updates a thread's properties.",
+        "operationId": "UpdateChatThread",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "The id of the thread to update.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "updateChatThreadRequest",
+            "description": "Request payload for updating a chat thread.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/UpdateChatThreadRequest"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Thread was successfully updated."
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Update chat thread topic": {
+            "$ref": "./examples/Threads_UpdateChatThreadTopic.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Threads"
+        ],
+        "summary": "Gets a chat thread.",
+        "operationId": "GetChatThread",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "Thread id to get.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The action returns a `Thread` resource.",
```

Comment:
We probably should remove "`" from the description as it is not necessarily going to be used in a markup.

Reply:
updated

Reply:
Thanks for fixing this. I just noticed there are few more with the "`" as well